easybuild/easyconfigs/d/deepTools/deepTools-3.1.2-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,61 @@
+
+easyblock = "PythonPackage"
```

Comment:
@ebirn It would make more sense to make this whole thing a `Bundle`, and install `deepTools` itself via `exts_list` as well, see the recent `matplotlib` easyconfigs